### ISSUE TITLE
ITM support

### ISSUE THREAD

**Issue by sourcebox** [knurling-rs/probe-run#270]
I'm currently using ITM for log messages via the `itm_logger` crate. This works with openocd, but I see no option to use this here. It seems that `probe-rs` has some support for it already.

**Comment by sourcebox** [comment]
Yes, I'm using an ST-Link via OpenOCD/GDB, then VS Code to dump the ITM messages.

**Comment by Urhengulas** [comment]
In this setup itm, would be an option, but rtt is also. Why do you choose itm over rtt?

**Comment by Urhengulas** [comment]
TIL. That's the problem if you only work with j-links 😅
